src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,40 @@
 
 class Animal {
   // write your code here
+
+  constructor(name) {
```

Comment:
```suggestion
  constructor(name) {
```
Better remove the comments and empty lines